### PR TITLE
#6234 #6239 JVM options improvements

### PR DESCRIPTION
**Pull request by rbidas**

### CODE REVIEW

File: beakerx/beakerx/environment.py [hunk 1]
```diff
@@ -14,20 +14,31 @@
 
 from os import environ, path
 from jupyter_core import paths
+import json
+
+sadsd = "" \
```
Comment (scottdraves) [inline beakerx/beakerx/environment.py]: what is this?

File: beakerx/beakerx/environment.py [hunk 2]
```diff
@@ -14,20 +14,31 @@
...
+        ""
+default_config = "{" \
```
Comment (scottdraves) [inline beakerx/beakerx/environment.py]: use triple quotes

File: beakerx/beakerx/environment.py (hunk 1 above)
Comment (rbidas) [reply beakerx/beakerx/environment.py]: removed

File: beakerx/beakerx/environment.py (hunk 2 above)
Comment (rbidas) [reply beakerx/beakerx/environment.py]: done

**Pull request merged**